app/src/sharedTest/java/org/oppia/android/app/testing/InputInteractionViewTestActivityTest.kt

```diff
@@ -816,13 +833,11 @@ class InputInteractionViewTestActivityTest {
   }
 
   @Test
-  // TODO(#973): Fix InputInteractionViewTestActivityTest
-  @Ignore
   fun testRatioInputInteractionView_withInputtedNegativeRatio_numberFormatErrorIsDisplayed() {
     ActivityScenario.launch(InputInteractionViewTestActivity::class.java)
     onView(withId(R.id.test_ratio_input_interaction_view))
```

Comment:
This test case is failing

Reply:
This is the error - 
```
androidx.test.espresso.base.DefaultFailureHandler$AssertionFailedWithCauseError: 'with string from resource id: <2131755343>' doesn't match the selected view.
Expected: with string from resource id: <2131755343>[ratio_error_invalid_format] value:
**** Please enter a valid ratio (e.g. 1:2 or 1:2:3). ****
     Got: "AppCompatTextView{id=2131296812, res-name=ratio_input_error, visibility=VISIBLE, width=888, height=96, has-focus=false, has-focusable=false, has-window-focus=true, is-clickable=false, is-enabled=true, is-focused=false, is-focusable=false, is-layout-requested=false, is-selected=false, layout-params=android.widget.LinearLayout$LayoutParams@7be72b1a, tag=null, root-is-layout-requested=false, has-input-connection=false, x=60.0, y=532.0, 
***  text=Please write a ratio that consists of digits separated by colons (e.g. 1:2 or 1:2:3). ***
, input-type=0, ime-target=false, has-links=false}"
```

I have highlighted the expected and actual texts with ****